core/subprojects/packagefiles/icu/meson.build

```diff
@@ -35,7 +35,7 @@ uconfig.set('U_ENABLE_DYLOAD', 0) # no DLL
 uconfig.set('U_CHECK_DYLOAD', 0) # no DLL
 uconfig.set('UCONFIG_NO_FILE_IO', 1)
 uconfig.set('UCONFIG_NO_LEGACY_CONVERSION', 1) # turn off file based codepage conversion
-uconfig.set('UCONFIG_NO_NORMALIZATION', 1) # TODO-LDML: may want this
+uconfig.set('UCONFIG_NO_NORMALIZATION', 0)
```

Comment:
Yeah. you think we might want this at zero? 

I do wonder why my build locally (mac) passed with this at 1.